release/rebuild_bom_files.sh

```diff
@@ -20,9 +20,7 @@ function check_usage {
 
 	_ARTIFACT_RC_VERSION="${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}"
 
-	_RELEASE_TOOL_DIR=$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")
-
-	lc_cd "${_RELEASE_TOOL_DIR}"
+	lc_cd $(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")
```

Comment:
@lucasmiranda0, can you please check if this same problem happens with `build_release` and `release_gold`?

Thanks

Reply:
@lucasmiranda0, this commit should be reverted because we must set the variable `_RELEASE_TOOL_DIR` otherwise the rebuild script will fail. This was the cause of the issue locally on my machine. Sorry about that.